protocol/x/clob/memclob/memclob.go

```diff
@@ -935,19 +935,6 @@ func (m *MemClobPriceTimePriority) ReplayOperations(
 				continue
 			}
 
-			if m.operationsToPropose.IsOrderRemovalInOperationsQueue(statefulOrderPlacement.Order.OrderId) {
-				// If an order removal for this order is found in the ops queue, then we should not attempt to
-				// place the order. Any generated matches would fail in DeliverTx as the order would be missing from
-				// state (due to the order removal being processed).
-				telemetry.IncrCounter(
-					1,
-					types.ModuleName,
-					metrics.ReplayOperations,
-					metrics.SkipStatefulReplayPlaceOrder,
-				)
-				continue
-			}
```

Comment:
Actually can we change this to an error log instead? I don't think this should ever happen and if it does it's an error.

Reply:
sure